### PR TITLE
566 story frontend routing and skeleton for lab 12

### PR DESCRIPTION
**Pull request by dm9965**
- [x] No discrepancies across browsers. Eg. chrome vs safari
- [x] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [x] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: .vscode/settings.json
Comment (heathermoses) [inline .vscode/settings.json:1]: This file shouldn't be committed

**Review by heathermoses — COMMENTED** [review]
I can't approve since I contributed but left one comment

**Review by heathermoses — COMMENTED** [review]
Good!

**Review by AndreasLc1103 — APPROVED** [review]
Lets get that bag

**Pull request merged**